tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -115,7 +116,11 @@ isl::union_set activeDomainPoints(
     return domain;
   }
 
-  for (auto anc : node->ancestors(root)) {
+  auto ancestors = node->ancestors(root);
+  if (inclusive) {
```

Comment:
This looks very fishy to me, see [here](https://github.com/facebookresearch/TensorComprehensions/blob/67b50561d913a526701388e8b42b045a2a9545d7/tc/core/polyhedral/schedule_tree.cc#L100), ancestors includes node (and actually checks for it as an invariant)

Reply:
See https://github.com/facebookresearch/TensorComprehensions/blob/67b50561d913a526701388e8b42b045a2a9545d7/tc/core/polyhedral/schedule_tree.cc#L104

Reply:
my bad misread the impl, sorry about that